terraform/dev.tfvars

```diff
@@ -1,8 +1,6 @@
 aws_account = "dev"
-system_dns_zone_id = "Z1QGLFML8EG6G7"
-system_dns_zone_name = "dev.cloudpipeline.digital"
+env_dns_zone_id = "Z1QGLFML8EG6G7"
```

Comment:
I agree with @alext, I think we should keep the naming "system" and "apps".

> I picked this one as it refers to entire environment not just root or system domain.

I don't quite understand this though. Aren't they the same thing, and different from the app domain?